src/Lean/Elab/BuiltinNotation.lean

```diff
@@ -131,7 +131,7 @@ At runtime, `msg` and the file position are printed to stderr unless the C
 function `lean_set_panic_messages(false)` has been executed before. If the C
 function `lean_set_exit_on_panic(true)` has been executed before, the process is
 then aborted. -/
-@[builtinTermElab panic] def elabPanic : TermElab := fun stx expectedType? => do
+@[builtinTermElab Lean.Parser.Term.panic] def elabPanic : TermElab := fun stx expectedType? => do
```

Comment:
@leodemoura minor regression where it accidentally resolved to `_root_.panic`, but since the change is purely internal, I don't think this is even worth a check whether the resolved constant is indeed a parser.